### ISSUE TITLE
Is this project still being maintained

### ISSUE THREAD

**Issue by Balajanovski** [conapps/dynamodb-cri#33]
This project perfectly fits the use case of my application.
However, it seems as though the current build is failing.
Is this project still being maintained?

**Comment by mxro** [comment]
Yes this looks pretty neat! Did you find another solution and therefore do not maintain this anymore? What would be the tradeoffs between this and DynamoDB Toolbox?
